packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -44,8 +44,33 @@
                     </fieldset>
                 </div>
 
+                <f-alert
```

Comment:
Personal preference potentially, but I'd abstract the notification/alerts out into their own `.vue` file (`notifications.vue`) that gets pulled in here, to keep the root component a bit cleaner.

Reply:
Can this be a new ticket?

Reply:
https://justeattakeaway.atlassian.net/browse/WCB-2524